reference/library/src/morpheus-master/sass/modules/tool/_menu.scss

```diff
@@ -66,6 +66,13 @@
 .listNav{
 	margin: 1em 0;
 	.inlineForm{
+		@extend .panel;
```

Comment:
Hi, I think this could get some incompatibilities with some tools. Class `linkNav`is used everywhere. If this fixes samigo I think should be on `_samigo.scss` file under its own namespace.

Anyway, I want to check this before saying yes or no.